gnomad/utils/sparse_mt.py

```diff
@@ -319,11 +322,18 @@ def get_as_info_expr(
     if "AS_SB" in agg_expr:
         agg_expr["AS_SB_TABLE"] = agg_expr.pop("AS_SB")
 
+    if alt_alleles_range_array not in mt.row or mt[
+        alt_alleles_range_array
+    ].dtype != hl.dtype("array<int32>"):
+        msg = f"'get_as_info_expr' expected a row field '{alt_alleles_range_array}' of type array<int32>"
+        logger.error(msg)
+        raise RuntimeError(msg)
```

Comment:
Nitpick, but `ValueError` seems like it would be slightly more appropriate here.
https://docs.python.org/3/library/exceptions.html#ValueError